crates/ironrdp-testsuite-core/tests/pdu/pointer.rs

```diff
@@ -5,6 +5,7 @@ use ironrdp_pdu::pointer::{
     PointerPositionAttribute,
 };
 
+#[expect(clippy::panic, reason = "panic is acceptable in tests")]
```

Comment:
suggestion: Move the #![allow(clippy::panic)] at the top of the main entry point instead